src/modules/scene/sagas.ts

```diff
@@ -280,3 +282,24 @@ function* handleSetGround(action: SetGroundAction) {
 
   yield put(provisionScene({ ...scene, components, entities, ground }))
 }
+
+function* handleFixCurrentScene() {
+  const scene: Scene = yield select(getCurrentScene)
+  const bounds: ReturnType<typeof getCurrentBounds> = yield select(getCurrentBounds)
+  const componentIdKeys = Object.keys(scene.components)
+  let components = { ...scene.components }
+
+  for (let key of componentIdKeys) {
+    const component = scene.components[key] as ComponentDefinition<ComponentType.Transform>
+    if (component.type === ComponentType.Transform && bounds) {
```

Comment:
instead of adding the `&& bounds` here, we could add an `if (!bounds) return` before the for loop